tests/reading-tests.md

```diff
@@ -0,0 +1,397 @@
+# Reading Tests
+
+## Goal
+
+The concepts about unit testing apply across programming languages! Everything we've learned about unit testing continues to hold true.
+
+- Unit tests are responsible for verifying the correctness of our code, usually on a detailed level such as how a function behaves
+- We regularly run unit tests as we code
+- We use failing test messages to determine how to move forward and pass the test
```

Comment:
This sounds like an observation specific to TDD. I say this because if I suddenly broke a test, I wouldn't phrase the process of figuring out what I broke as moving forward to pass the test. I would say that I was resolving a problem. This more forward looking perspective sounds like we wrote a test that encodes a specification and we are trying to write an implementation to make it pass.

Under that assumption, it sounds better to me to provide a little of that context here.
```suggestion
- When using tests to guide our development work, we use failing test messages to determine how to move forward and pass the test
```